### PR TITLE
Make glasbey installable

### PR DESCRIPTION
**Pull request by ghost**
This request will allow `glasbey` to installed with `pip` thus.

* pip install `git+https://github.com/taketwo/glasbey.git`
* pip install `glasbey[progressbar]@git+https://github.com/taketwo/glasbey.git`
* pip install `glasbey[view_palette]@git+https://github.com/taketwo/glasbey.git`
* pip install `glasbey[progressbar,view_palette]@git+https://github.com/taketwo/glasbey.git`

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[metadata]
+name = glasbey
+version = 0.0.0
```
Comment (taketwo) [inline setup.cfg:3]: Let's make it `1.0.0.` :)

**Review by taketwo — COMMENTED** [review]
That's a great contribution, thank you! I'll give it a try soon.

**Comment by taketwo** [comment]
If I understand right, this installs the package as a library, i.e. after installation the user may fire up Python interpreter and `import glasbey`. Would it a lot of work to also expose `glasbey.py` as an app? Such that after installation the user may also run `glasbey` in terminal?

**Comment by JDawson-Camlin** [comment]
After `glasbey` is installed shell commands such as the following work
```shell
python -m glasbey --no-black 30 out
python -m view_palette -h
python -m view_palette out
```

**Pull request merged**